### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by sheylaPozo**
- Use basic JavaScript syntax.
- Use JavaScript to manipulate DOM elements.
- Use JavaScript events.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+menu.appendChild(menuList);
+
+const portfolio = document.createElement("li");
+portfolio.className = "portfolio";
+const portfolioLink = document.createElement("a");
+portfolioLink.className = "mobile-link";
+portfolioLink.textContent = "Portfolio";
+portfolioLink.href = "#portfolio";
+portfolio.appendChild(portfolioLink);
+menuList.appendChild(portfolio);
```
Comment (Temmarie) [inline script.js:21]: - [x] Good job working on the mobile menu, the general concepts has been met, but there's a catch.
- Upon running your project on the live server it shows up with the mobile menu already open, same thing goes for the Desktop screens. Kindly fix this.
  See the images below.
_Mobile screens_
image
_Desktop screens_
image

**Review by Temmarie — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED! :recycle:

### Hi @sheylaPozo    ! :wave: :wave:

#### You have done really well in implementing the requirements.
- You used proper GitHub Flow ✔️ 
- Your linter checks are all passing ✔️ 
- Your README and PR message are descriptive ✔️ 

 However, there are changes required for you to make in order for this project to be approved. 
- - -

### **Required Changes** ♻️
_See the inline comments below for the required changes_


### Optional Changes 👌
Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better.
- N/A

_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

**_"Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."_**
- - - 
**Note that you still have 2 more reviews as written in the [Code Reviews Limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561), however, if you feel this review is unfair, you can request a second opinion using [this](https://airtable.com/shrQAqnBwek5a0O0s) form.**

__Kindly make the necessary changes and submit another code review request.__

File: script.js (hunk 1 above)
Comment (sheylaPozo) [reply script.js:21]: thank you, just fixed it.

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**